game_eggs/steamcmd_servers/ark_survival_ascended/README.md

```diff
@@ -0,0 +1,23 @@
+# Ark: Survival Ascended
```

Comment:
ARK should be capitalized.

Reply:
arkse is writting same way here - would keep it consistent?

Reply:
I hadn't noticed. I'd just fix it there too considering it's the branded stylization of the game's name but otherwise whatever.